### PR TITLE
Rename SUB_UNIVERSE to SUB_KIND and add examples.

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

File: src/redprl/atomic_judgment.sig [hunk 1]
```diff
@@ -41,12 +41,12 @@ struct
       *)
      | SUB_TYPE of (abt * abt)
 
-     (* `SUB_UNIVERSE (a, k)`
-      *   `a` is a sub-universe of U^`k`.
+     (* `SUB_KIND (a, k)`
+      *   `a` is a sub-universe of the universe of `k` types at the omega level.
```
Comment (favonia) [inline src/redprl/atomic_judgment.sig]: @jonsterling Please check that `SUB_KIND` is the proper name for this semantics.

Comment (jonsterling) [reply src/redprl/atomic_judgment.sig]: I think it mostly makes sense. The only thing that is a bit weird is I would expect subkind to have two kinds as its arguments. Would it make more sense to say `HAS_KIND`? That is, the universe `a` can be regarded as having the kind `k`.

Comment (favonia) [reply src/redprl/atomic_judgment.sig]: Wouldn't that be misunderstood as the universe itself being of kind `k`?

Comment (jonsterling) [reply src/redprl/atomic_judgment.sig]: @favonia Haha, you are right 😉

**Comment by favonia** [comment]
Any other comments? I would like to merge this because of the conflicts.

**Comment by jonsterling** [comment]
go for it

**Pull request merged**